#!/usr/bin/env python
PACKAGE = "image_proc_chain"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enable", bool_t, 0, "Enable this processor", True)

gen.add("iteration_count", int_t, 0, "How many process pass through", 1,  1, 10)
gen.add("kernel_size", int_t, 0, "Width and height of kernel", 3,  3, 50)

exit(gen.generate(PACKAGE, "image_proc_chain", "Dilation"))
